src/librustdoc/clean/mod.rs

```diff
@@ -169,11 +169,19 @@ impl Clean<ExternalCrate> for CrateNum {
                 for attr in attrs.lists(sym::doc) {
                     if let Some(v) = attr.value_str() {
                         if attr.has_name(sym::keyword) {
-                            if v.is_doc_keyword() {
```

Comment:
Could you remove `fn is_doc_keyword` as well? This was its only use.

Reply:
Sure!